tests/publiccloud/create_aistack_env.pm

```diff
@@ -45,8 +49,9 @@ sub install_dependency_package {
 
     # rke2 activation section
     script_run("curl -sSL $rke2_url -o ./install_rke2.sh && chmod 775 ./install_rke2.sh");
-    script_run("sh ./install_rke2.sh");
+    script_run("sh ./install_rke2.sh | sh -");
     script_run("echo 'export PATH=\$PATH:/opt/rke2/bin' >> ~/.bashrc");
```

Comment:
IMHO All above shell runs, untill here, should be done as `assert_...`, being a failure critical. Avoiding uneeded asserts, let's instead apply it where makes sense.

Reply:
fixed